docs/content/recipes/common/replace-crio-runtime.md

```diff
@@ -0,0 +1,261 @@
+# Replacing the Worker Node's CRI-O Runtime
+
+Starting with OpenShift version 4.18, the Machine Config Operator (MCO) will switch the default runtime from `runc` to `crun`, simplying so much the internals for the simplicity, crun is a simplified implementation of runc written in C. You can find more information about this change [here](https://github.com/containers/crun?tab=readme-ov-file#documentation).
```

Comment:
simplying so much the internals for the simplicity - I don't understand what's trying to be conveyed here.

Reply:
basically tell the people what's happening under the hood without enter in much details that maybe are not relevant for them.

Reply:
It doesn't make sense to me structure wise atm. Could you change it to something like `simplifying much of the internals`?